examples/split_deck.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import sys
+from sunbeam.deck import Deck
+from sunbeam      import action
+
+def split_deck( file_name ):
+    recovery = ('PARSE_RANDOM_SLASH', action.ignore)
+    deck = Deck.parse( file_name, recovery=recovery )
+
+    sections = [ deck.select[ 'RUNSPEC'  ],
```

Comment:
I would prefer—since this is an example, it ought to be perfect—that you write something like
```python
sections_ = ('RUNSPEC', 'GRID', 'EDIT', 'PROPS', 'REGIONS', 'SOLUTION',
             'SUMMARY', 'SCHEDULE')
sections = [deck.select(section) for section in _sections]
```
and the run `yapf` on this file.